Sprint-1/errors/3.js

```diff
@@ -1,9 +1,13 @@
-const cardNumber = 4533787178994213;
+const cardNumber = "4533787178994213";
```

Comment:
If you cannot modify this statement `const cardNumber = 4533787178994213;`
(that is, keep the variable's value unchanged), 
how would you modify the code so that it can still extract the last 4 digits from its value.

Reply:
the type of variable needs to string for .slice() method to work.